crates/subspace-farmer/src/bin/subspace-farmer/commands/farm/dsn.rs

```diff
@@ -277,6 +280,8 @@ pub(super) fn configure_dsn(
             }),
         ],
         provider_storage: farmer_provider_storage,
+        max_established_outgoing_connections: MAX_ESTABLISHED_OUTGOING_CONNECTIONS,
+        max_pending_outgoing_connections: MAX_PENDING_OUTGOING_CONNECTIONS,
```

Comment:
Why is this necessary? I thought 50 was a reasonable default, why do we need higher on the farmer specifically? Also with both bootstrap-node and subspace-node we have ability to configure these using CLI arguments, so I don't understand why not using the same approach here too.

Reply:
Discussed it offline. We introduce CLI connection args for farmers.